test/unittests/commands/excel2json/test_properties.py

```diff
@@ -432,20 +389,46 @@ def test__do_property_excel_compliance(self) -> None:
                 "gui_attributes": ["size: 32, maxlength: 128", pd.NA, pd.NA, pd.NA, pd.NA, pd.NA, pd.NA, pd.NA],
             }
         )
-        with self.assertRaisesRegex(
-            UserError,
-            r"The file 'Test' is missing values in the following rows\:\n"
-            r"\- Column 'name' Row Number\(s\)\: \[9\]\n"
-            r"\- Column 'super' Row Number\(s\)\: \[2, 4, 9\]\n"
-            r"\- Column 'object' Row Number\(s\)\: \[5, 9\]\n"
-            r"\- Column 'gui_element' Row Number\(s\)\: \[6, 8, 9\]\n"
-            r"\- Column 'label' Row Number\(s\)\: \[3, 8\]\n"
-            r"\- Column 'gui_attributes' Row Number\(s\)\: \[7\]",
-        ):
-            e2j._do_property_excel_compliance(df=original_df, excelfile="Test")
+        expected_msg = re.escape(
+            (
+                "The excel file 'Test' has some problems:\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: name\n"
+                "\tRow(s):\n"
+                "\t- 9\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: super\n"
+                "\tRow(s):\n"
+                "\t- 2\n"
+                "\t- 4\n"
+                "\t- 9\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: object\n"
+                "\tRow(s):\n"
+                "\t- 5\n"
+                "\t- 9\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: gui_element\n"
+                "\tRow(s):\n"
+                "\t- 6\n"
+                "\t- 8\n"
+                "\t- 9\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: label\n"
+                "\tRow(s):\n"
+                "\t- 3\n"
+                "\t- 8\n"
+                "There are missing values in a column that must not be empty:\n"
+                "\tColumn: gui_attributes\n"
+                "\tRow(s):\n"
+                "\t- 7"
```

Comment:
does this really have to be checked with a regex? looks like you know the literal string, so a simple equality check should be possible, right? And then, a multi-line string would probably be more readable

Reply:
I will check with the equality, but regarding the multi-line strings, I have always had problems in the past with matching them, as the implicit linebreaks etc. are sometimes not recognised as equal to the explicit ones...

Reply:
now that I think about it, that makes sense - if you indent multiline-strings, you get extra whitespace. (in Scala we can strip this whitespace, which is very handy...)